package.json

```diff
@@ -12,7 +12,9 @@
   "scripts": {
     "dev": "wp-scripts start",
     "build": "wp-scripts build",
-    "readme": "grunt wp_readme_to_markdown"
+    "readme": "grunt wp_readme_to_markdown",
+	"env-start": "wp-env start && wp-env run cli wp rewrite structure '/%year%/%monthnum%/%postname%/'",
```

Comment:
@kraftbj can you use the same indents as the rest of the file (4 spaces)? I am also fine in switching to tabs, but it should be consistent.

Reply:
Will do. #BlameIDE

Reply:
940b1e0